appinventor/appengine/src/com/google/appinventor/client/explorer/commands/DeleteFileCommand.java

```diff
@@ -36,6 +70,158 @@ public boolean willCallExecuteNextCommand() {
 
   @Override
   public void execute(final ProjectNode node) {
+	  if (node instanceof YoungAndroidSourceNode) {
+  		new NewFormDialog((YoungAndroidSourceNode) node).center();
+  		
+	  } 
+	  else {
+  		  executionFailedOrCanceled();
+  		  throw new IllegalArgumentException("node must be a YoungAndroidProjectNode");
+	  }
+  }
+
+  /**
+   * Shows a confirmation dialog.
+   *
+   * @return {@code true} if the delete file command should be executed or
+   *         {@code false} if it should be canceled
+   */
+  protected boolean deleteConfirmation() {
+    return Window.confirm(MESSAGES.reallyDeleteFile());
+  }
+  
+  private class NewFormDialog extends DialogBox {
```

Comment:
This class should have a more meaningful name. We're deleting a form, not adding one.